Lottery.java

```diff
@@ -0,0 +1,11 @@
+import java.util.Random;
+public class Lottery {
+    public static ball getRandomBall(){
+        Random random = new Random();
+        int numb = random.nextInt(100);
+        ColorSupplier ColorsSupplier = new ColorSupplier();
+        Colors gege = ColorsSupplier.getRandomColor();
+        ball ball = new ball(numb, gege);

```

Comment:
The class name `ball` should be capitalized as `Ball` to follow Java naming conventions for classes.